tests/system/libraries/NvdaLib.py

```diff
@@ -200,7 +200,7 @@ def _startNVDAInstallerProcess(self):
 		)
 		return handle
 
-	def _connectToRemoteServer(self, connectionTimeoutSecs=10):
+	def _connectToRemoteServer(self, connectionTimeoutSecs=15):

```

Comment:
```suggestion
	def _connectToRemoteServer(self, connectionTimeoutSecs: int = 15) -> HWNDVal::
```

Reply:
Was this intended somewhere else? This method doesn't return a value.

Reply:
I misread this function

```suggestion
	def _connectToRemoteServer(self, connectionTimeoutSecs: int = 15) -> None:
```